reflex/components/datadisplay/logo.py

```diff
@@ -0,0 +1,21 @@
+from reflex.components.component import Component
+import reflex as rx
+
+class Logo(Component):
+
+    @classmethod
+    def create(cls, **props,):
+        return rx.center(
+        rx.link(
+            rx.hstack(
+                "Built with ",
+                rx.image(src="https://raw.githubusercontent.com/reflex-dev/reflex-web/main/assets/Reflex.svg"),
```

Comment:
We can do `rx.color_mode_cond` and show the white logo in dark mode: https://github.com/reflex-dev/reflex-web/blob/main/assets/Reflex_white.svg

Reply:
done